test-infra/camel-test-infra-kafka/src/test/java/org/apache/camel/test/infra/services/kafka/StrimziService.java

```diff
@@ -24,8 +24,11 @@
 
 public class StrimziService implements KafkaService {
     private static final Logger LOG = LoggerFactory.getLogger(StrimziService.class);
+    
```

Comment:
Sourcecheck complains about this line with one tab/spaces. You can run `mvn org.apache.camel:camel-format-plugin:format` in `test-infra/camel-test-infra-kafka` and commit.

Reply:
Thanks @bedlaj!

Reply:
Yeap. That was it. Good catch!